src/graphics/sdl/sdl_graphics.cpp

```
@@ -136,27 +153,22 @@ namespace canyon::graphics::sdl {
-        rmask = 0x000000ff;
-        gmask = 0x0000ff00;
-        bmask = 0x00ff0000;
-        amask = 0xff000000;
-#endif
-
-        int width;
-        int height;
-        SDL_GetRendererOutputSize(m_surfaceContext.GetRenderer(), &width, &height);
-
-        SurfaceRef surface = CreateSurfaceRef(SDL_CreateRGBSurface(0, width, height, 32, rmask, gmask, bmask, amask));
+    void Graphics::DrawToPNG(IImage& image, std::filesystem::path const& path) {
```

Clang (14.0.6)

[error] method 'DrawToPNG' can be made static

(readability-convert-member-functions-to-static,-warnings-as-errors)